src/main/java/github/scarsz/discordsrv/hooks/chat/VentureChatHook.java

```diff
@@ -18,105 +18,155 @@
 
 package github.scarsz.discordsrv.hooks.chat;
 
+import dev.vankka.mcdiscordreserializer.discord.DiscordSerializer;
 import dev.vankka.mcdiscordreserializer.minecraft.MinecraftSerializer;
 import github.scarsz.discordsrv.DiscordSRV;
+import github.scarsz.discordsrv.util.DiscordUtil;
 import github.scarsz.discordsrv.util.LangUtil;
 import github.scarsz.discordsrv.util.PlayerUtil;
 import github.scarsz.discordsrv.util.PluginUtil;
+import github.scarsz.discordsrv.util.TimeUtil;
 import mineverse.Aust1n46.chat.MineverseChat;
-import mineverse.Aust1n46.chat.api.MineverseChatAPI;
 import mineverse.Aust1n46.chat.api.MineverseChatPlayer;
+import mineverse.Aust1n46.chat.api.events.VentureChatEvent;
 import mineverse.Aust1n46.chat.channel.ChatChannel;
+import mineverse.Aust1n46.chat.utilities.Format;
 import net.kyori.text.adapter.bukkit.TextAdapter;
+import net.kyori.text.serializer.legacy.LegacyComponentSerializer;
+
+import org.apache.commons.lang3.StringUtils;
 import org.bukkit.ChatColor;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.EventPriority;
-import org.bukkit.event.player.AsyncPlayerChatEvent;
 import org.bukkit.plugin.Plugin;
 
+import com.comphenix.protocol.events.PacketContainer;
```

Comment:
ProtocolLib would be **required** if this is added, is there a alternative?

Reply:
Would only be required if VentureChat is in use, and VentureChat requires it.  I could change around the Format API but any plugin using VentureChat's API will always have access to ProtocolLib since I require it.

Reply:
In that case it'll need to be added to the pom, i'll get ProtocolLib uploaded onto nexus.scarsz.me for this